### PR TITLE
Myths and mysteries

### PR DESCRIPTION
**Pull request by Tiritirimatangi**
Quite a few new patrols all focused on making patrols for the story and lore keeper skills. Full devlog:

Myths and mysteries

gen_train_adultsonly_notlikethatyoumoron1
gen_train_adultsonly_notlikethatyoumoron2

gen_train_furiousatwhiskergrove_storylorespeakerteacherlocked1
gen_train_firestarfable_storylorelocked1
gen_train_silverpeltfable_storylorelocked1
gen_train_powerofthreefable_storylorelocked1
    gen_train_constellations_storylocked1
        gen_train_solreference_lorelocked1

bch_med_greenleaf_gatheringelder_lorehealingstorylocked1
bch_med_leaffall_gatheringelder_lorehealingstorylocked1
bch_med_newleaf_gatheringelder_lorehealingstorylocked1
bch_med_greenleaf_daisy_lorestoryhealinglocked1
bch_med_leaffall_daisy_lorestoryhealinglocked1
bch_med_newleaf_daisy_lorestoryhealinglocked1
bch_med_greenleaf_juniper_lorestoryhealerlocked1
bch_med_leaffall_juniper_lorestoryhealerlocked1
bch_med_newleaf_juniper_lorestoryhealerlocked1
bch_med_greenleaf_lungwort_storylorehealerlocked1
bch_med_leaffall_lungwort_storylorehealerlocked1
bch_med_newleaf_lungwort_storylorehealerlocked1
bch_med_greenleaf_mallow_storylorehealerlocked1
bch_med_leaffall_mallow_storylorehealerlocked1
bch_med_newleaf_mallow_storylorehealerlocked1
bch_med_greenleaf_marigold_storylorehealerlocked1
bch_med_leaffall_marigold_storylorehealerlocked1
bch_med_newleaf_marigold_storylorehealerlocked1
bch_med_greenleaf_oakleaves_storylorehealerlocked1
bch_med_leaffall_oakleaves_storylorehealerlocked1
bch_med_newleaf_oakleaves_storylorehealerlocked1
bch_med_greenleaf_ragwort_storylorehealerlocked1
bch_med_leaffall_ragwort_storylorehealerlocked1
bch_med_newleaf_ragwort_storylorehealerlocked1
bch_med_greenleaf_raspberry_storylorehealerlocked1
bch_med_leaffall_raspberry_storylorehealerlocked1
bch_med_newleaf_raspberry_storylorehealerlocked1
bch_med_greenleaf_tansy_storylorehealerlocked1
bch_med_leaffall_tansy_storylorehealerlocked1
bch_med_newleaf_tansy_storylorehealerlocked1
bch_med_greenleaf_thyme_risingclanref_storylorehealerlocked1 this one needs playtesting love
bch_med_leaffall_thyme_risingclanref_storylorehealerlocked1
bch_med_newleaf_thyme_risingclanref_storylorehealerlocked1

fst_med_greenleaf_daisy_lorestoryhealinglocked1
fst_med_greenleaf_gatheringelder_lorehealingstorylocked1
fst_med_greenleaf_juniper_lorestoryhealerlocked1
fst_med_greenleaf_lungwort_storylorehealerlocked1
fst_med_greenleaf_mallow_storylorehealerlocked1
fst_med_greenleaf_marigold_storylorehealerlocked1
fst_med_greenleaf_oakleaves_storylorehealerlocked1
fst_med_greenleaf_ragwort_storylorehealerlocked1
fst_med_greenleaf_raspberry_storylorehealerlocked1
fst_med_greenleaf_tansy_storylorehealerlocked1
fst_med_greenleaf_thyme_risingclanref_storylorehealerlocked1

fst_med_leaffall_daisy_lorestoryhealinglocked1
fst_med_leaffall_gatheringelder_lorehealingstorylocked1
fst_med_leaffall_juniper_lorestoryhealerlocked1
fst_med_leaffall_lungwort_storylorehealerlocked1
fst_med_leaffall_mallow_storylorehealerlocked1
fst_med_leaffall_marigold_storylorehealerlocked1
fst_med_leaffall_oakleaves_storylorehealerlocked1
fst_med_leaffall_ragwort_storylorehealerlocked1
fst_med_leaffall_raspberry_storylorehealerlocked1
fst_med_leaffall_tansy_storylorehealerlocked1
fst_med_leaffall_thyme_risingclanref_storylorehealerlocked1

fst_med_newleaf_daisy_lorestoryhealinglocked1
fst_med_newleaf_gatheringelder_lorehealingstorylocked1
fst_med_newleaf_juniper_lorestoryhealerlocked1
fst_med_newleaf_lungwort_storylorehealerlocked1
fst_med_newleaf_mallow_storylorehealerlocked1
fst_med_newleaf_marigold_storylorehealerlocked1
fst_med_newleaf_oakleaves_storylorehealerlocked1
fst_med_newleaf_ragwort_storylorehealerlocked1
fst_med_newleaf_raspberry_storylorehealerlocked1
fst_med_newleaf_tansy_storylorehealerlocked1
fst_med_newleaf_thyme_risingclanref_storylorehealerlocked1

dst_med_leafbare_daisy_lorestoryhealinglocked1
dst_med_leafbare_gatheringelder_lorehealingstorylocked1
dst_med_leafbare_juniper_lorestoryhealerlocked1
dst_med_leafbare_lungwort_storylorehealerlocked1
dst_med_leafbare_mallow_storylorehealerlocked1
dst_med_leafbare_marigold_storylorehealerlocked1
dst_med_leafbare_oakleaves_storylorehealerlocked1
dst_med_leafbare_ragwort_storylorehealerlocked1
dst_med_leafbare_raspberry_storylorehealerlocked1
dst_med_leafbare_tansy_storylorehealerlocked1
dst_med_leafbare_thyme_risingclanref_storylorehealerlocked1

dst_med_leaffall_daisy_lorestoryhealinglocked1
dst_med_leaffall_gatheringelder_lorehealingstorylocked1
dst_med_leaffall_juniper_lorestoryhealerlocked1
dst_med_leaffall_lungwort_storylorehealerlocked1
dst_med_leaffall_mallow_storylorehealerlocked1
dst_med_leaffall_marigold_storylorehealerlocked1
dst_med_leaffall_oakleaves_storylorehealerlocked1
dst_med_leaffall_ragwort_storylorehealerlocked1
dst_med_leaffall_raspberry_storylorehealerlocked1
dst_med_leaffall_tansy_storylorehealerlocked1
dst_med_leaffall_thyme_risingclanref_storylorehealerlocked1

dst_med_newleaf_daisy_lorestoryhealinglocked1
dst_med_newleaf_gatheringelder_lorehealingstorylocked1
dst_med_newleaf_juniper_lorestoryhealerlocked1
dst_med_newleaf_lungwort_storylorehealerlocked1
dst_med_newleaf_mallow_storylorehealerlocked1
dst_med_newleaf_marigold_storylorehealerlocked1
dst_med_newleaf_oakleaves_storylorehealerlocked1
dst_med_newleaf_ragwort_storylorehealerlocked1
dst_med_newleaf_raspberry_storylorehealerlocked1
dst_med_newleaf_tansy_storylorehealerlocked1
dst_med_newleaf_thyme_risingclanref_storylorehealerlocked1

mtn_med_greenleaf_daisy_lorestoryhealinglocked1
mtn_med_greenleaf_gatheringelder_lorehealingstorylocked1
mtn_med_greenleaf_juniper_lorestoryhealerlocked1
mtn_med_greenleaf_lungwort_storylorehealerlocked1
mtn_med_greenleaf_mallow_storylorehealerlocked1
mtn_med_greenleaf_marigold_storylorehealerlocked1
mtn_med_greenleaf_oakleaves_storylorehealerlocked1
mtn_med_greenleaf_ragwort_storylorehealerlocked1
mtn_med_greenleaf_raspberry_storylorehealerlocked1
mtn_med_greenleaf_tansy_storylorehealerlocked1
mtn_med_greenleaf_thyme_risingclanref_storylorehealerlocked1

mtn_med_leaffall_daisy_lorestoryhealinglocked1
mtn_med_leaffall_gatheringelder_lorehealingstorylocked1
mtn_med_leaffall_juniper_lorestoryhealerlocked1
mtn_med_leaffall_lungwort_storylorehealerlocked1
mtn_med_leaffall_mallow_storylorehealerlocked1
mtn_med_leaffall_marigold_storylorehealerlocked1
mtn_med_leaffall_oakleaves_storylorehealerlocked1
mtn_med_leaffall_ragwort_storylorehealerlocked1
mtn_med_leaffall_raspberry_storylorehealerlocked1
mtn_med_leaffall_tansy_storylorehealerlocked1
mtn_med_leaffall_thyme_risingclanref_storylorehealerlocked1

mtn_med_newleaf_daisy_lorestoryhealinglocked1
mtn_med_newleaf_gatheringelder_lorehealingstorylocked1
mtn_med_newleaf_juniper_lorestoryhealerlocked1
mtn_med_newleaf_lungwort_storylorehealerlocked1
mtn_med_newleaf_mallow_storylorehealerlocked1
mtn_med_newleaf_marigold_storylorehealerlocked1
mtn_med_newleaf_oakleaves_storylorehealerlocked1
mtn_med_newleaf_ragwort_storylorehealerlocked1
mtn_med_newleaf_raspberry_storylorehealerlocked1
mtn_med_newleaf_tansy_storylorehealerlocked1
mtn_med_newleaf_thyme_risingclanref_storylorehealerlocked1

pln_med_greenleaf_daisy_lorestoryhealinglocked1
pln_med_greenleaf_gatheringelder_lorehealingstorylocked1
pln_med_greenleaf_juniper_lorestoryhealerlocked1
pln_med_greenleaf_lungwort_storylorehealerlocked1
pln_med_greenleaf_mallow_storylorehealerlocked1
pln_med_greenleaf_marigold_storylorehealerlocked1
pln_med_greenleaf_oakleaves_storylorehealerlocked1
pln_med_greenleaf_ragwort_storylorehealerlocked1
pln_med_greenleaf_raspberry_storylorehealerlocked1
pln_med_greenleaf_tansy_storylorehealerlocked1
pln_med_greenleaf_thyme_risingclanref_storylorehealerlocked1

pln_med_leaffall_daisy_lorestoryhealinglocked1
pln_med_leaffall_gatheringelder_lorehealingstorylocked1
pln_med_leaffall_juniper_lorestoryhealerlocked1
pln_med_leaffall_lungwort_storylorehealerlocked1
pln_med_leaffall_mallow_storylorehealerlocked1
pln_med_leaffall_marigold_storylorehealerlocked1
pln_med_leaffall_oakleaves_storylorehealerlocked1
pln_med_leaffall_ragwort_storylorehealerlocked1
pln_med_leaffall_raspberry_storylorehealerlocked1
pln_med_leaffall_tansy_storylorehealerlocked1
pln_med_leaffall_thyme_risingclanref_storylorehealerlocked1

pln_med_newleaf_daisy_lorestoryhealinglocked1
pln_med_newleaf_gatheringelder_lorehealingstorylocked1
pln_med_newleaf_juniper_lorestoryhealerlocked1
pln_med_newleaf_lungwort_storylorehealerlocked1
pln_med_newleaf_mallow_storylorehealerlocked1
pln_med_newleaf_marigold_storylorehealerlocked1
pln_med_newleaf_oakleaves_storylorehealerlocked1
pln_med_newleaf_ragwort_storylorehealerlocked1
pln_med_newleaf_raspberry_storylorehealerlocked1
pln_med_newleaf_tansy_storylorehealerlocked1
pln_med_newleaf_thyme_risingclanref_storylorehealerlocked1

wtlnd_med_greenleaf_daisy_lorestoryhealinglocked1
wtlnd_med_greenleaf_gatheringelder_lorehealingstorylocked1
wtlnd_med_greenleaf_juniper_lorestoryhealerlocked1
wtlnd_med_greenleaf_lungwort_storylorehealerlocked1
wtlnd_med_greenleaf_mallow_storylorehealerlocked1
wtlnd_med_greenleaf_marigold_storylorehealerlocked1
wtlnd_med_greenleaf_oakleaves_storylorehealerlocked1
wtlnd_med_greenleaf_ragwort_storylorehealerlocked1
wtlnd_med_greenleaf_raspberry_storylorehealerlocked1
wtlnd_med_greenleaf_tansy_storylorehealerlocked1
wtlnd_med_greenleaf_thyme_risingclanref_storylorehealerlocked1

wtlnd_med_leaffall_daisy_lorestoryhealinglocked1
wtlnd_med_leaffall_gatheringelder_lorehealingstorylocked1
wtlnd_med_leaffall_juniper_lorestoryhealerlocked1
wtlnd_med_leaffall_lungwort_storylorehealerlocked1
wtlnd_med_leaffall_mallow_storylorehealerlocked1
wtlnd_med_leaffall_marigold_storylorehealerlocked1
wtlnd_med_leaffall_oakleaves_storylorehealerlocked1
wtlnd_med_leaffall_ragwort_storylorehealerlocked1
wtlnd_med_leaffall_raspberry_storylorehealerlocked1
wtlnd_med_leaffall_tansy_storylorehealerlocked1
wtlnd_med_leaffall_thyme_risingclanref_storylorehealerlocked1

wtlnd_med_newleaf_daisy_lorestoryhealinglocked1
wtlnd_med_newleaf_gatheringelder_lorehealingstorylocked1
wtlnd_med_newleaf_juniper_lorestoryhealerlocked1
wtlnd_med_newleaf_lungwort_storylorehealerlocked1
wtlnd_med_newleaf_mallow_storylorehealerlocked1
wtlnd_med_newleaf_marigold_storylorehealerlocked1
wtlnd_med_newleaf_oakleaves_storylorehealerlocked1
wtlnd_med_newleaf_ragwort_storylorehealerlocked1
wtlnd_med_newleaf_raspberry_storylorehealerlocked1
wtlnd_med_newleaf_tansy_storylorehealerlocked1
wtlnd_med_newleaf_thyme_risingclanref_storylorehealerlocked1

mtn_train_leafbare_darkforestfable_storylorelocked1
mtn_train_leaffall_darkforestfable_storylorelocked1
mtn_train_leafbare_warmthoftheclan_storylorelocked1
mtn_train_leafbare_avalanche_storylorecleverinsightteacherlocked1
mtn_train_newleaf_hibernation_storylorelocked1
    mtn_train_mountainlion_storylocked1
    mtn_train_leaffall_eclipsemyth_storylocked1
    mtn_train_greenleaf_sunsetmyth_storylocked1
        mtn_hunt_wareprey_lorelocked1
        mtn_train_leafbare_thecurseofleafbare_lorelocked1

pln_train_thesunandmoonsiblings_storylocked1
pln_train_thelongjourneyhome_storylorelocked1
pln_train_newleaf_circleoflife_storylorelocked1
pln_train_greenleaf_lackofshade_storyloreteacherlocked1
pln_train_leaffall_locustswarm_storylorelocked1
    pln_train_beautyandthebeast_storylocked1
    pln_med_leaffall_raspberriescolourmyth_storylocked1
    pln_med_greenleaf_raspberriescolourmyth_storylocked1
        pln_hunt_wareprey_lorelocked1
        pln_train_leafbare_thecurseofleafbare_lorelocked1
        pln_train_thunderpathtraining_loreteacherinsightfullocked1

fst_train_newleaf_mapleshadereference_storylorelocked1
fst_hunt_risingclangift_storylorelocked1
fst_hunt_leafbare_dewclangift_storylorelocked1
fst_hunt_leafbare_kingfisher_storylorehunterclimberteacherlocked1
    fst_train_leafbare_snowmyth_storylocked1
    fst_train_newleaf_crowrock_storylocked1
    fst_train_leaffall_eclipsemyth_storylocked1
    fst_med_leafbare_yewcolour_storylocked1
    fst_med_leaffall_yewcolour_storylocked1
    fst_train_leafbare_underwaterpanthermyth_storylocked1
        fst_train_wareprey_lorelocked1
        fst_train_leafbare_thecurseofleafbare_lorelocked1
        fst_bord_risingclanref_allies_lorelocked1
        fst_bord_risingclanref_neutral_lorelocked1

dst_train_lizardsunworship_storylorelocked1 
dst_train_greenleaf_flippedseasons_storylorelocked1
dst_train_leafbare_flippedseasons_storylorelocked1
dst_train_kheprithescarab_storylorelocked1
dst_train_fossilsincanyons_storylorelocked1
    dst_train_wilecoyote_storylocked1
    dst_train_chupacabra_storylocked1
    dst_train_leafbare_dreamtime_storylocked1
        dst_hunt_wareprey_lorelocked1
        dst_train_thewaterwars_lorelocked1

bch_train_leafbare_stormfables_storylorelocked1
bch_train_leaffall_lifeguarding_storyloreswimmerlocked1
bch_train_themoonandthetides_storylorelocked1
    bch_train_ottercreationmyth_storylocked1
    bch_train_leafbare_icebergmyth_storylocked1
    bch_train_mauifishing_storylocked1
    bch_train_mauislowingthesun_storylocked1
        bch_train_greenleaf_twolegmigration_lorelocked1
        bch_hunt_wareprey_lorelocked1
        bch_train_foxdawn_lorelocked1
        bch_train_newleaf_keepyourfooting_loreteacherswimmerrunnerlocked1
            bch_train_greenleaf_dryhot1

wtlnd_train_thegreedyravenandthesnake_storylorelocked1
wtlnd_hunt_leafbare_kingfisher_storylorehunterclimberteacherlocked1
wtlnd_train_bogbodies_storylorelocked1
wtlnd_train_glowingmushrooms_apprentices_storylorelocked1
    wtlnd_train_leafbare_serpentmyth_storylocked1
    wtlnd_train_leaffall_eclipsemyth_storylocked1
    wtlnd_train_moosemyth_storylocked1
        wtlnd_hunt_wareprey_lorelocked1
        wtlnd_train_thechasedones_lorelocked1
            wtlnd_train_greenleaf_humidhot1

### CODE REVIEW

File: resources/dicts/patrols/beach/med/greenleaf.json [hunk 1]
```diff
@@ -2719,6 +2894,71 @@
...
+        "tags": [],
+        "patrol_art": "gen_bord_story",
+        "min_cats": 3,
+        "max_cats": 6,
+        "min_max_status": {
+            "healer cats": [1, 6]
+        },
+        "weight": 20,
+        "pl_skill_constraint": ["LORE,1", "STORY,1", "HEALER,1"],
+        "intro_text": "There's an old saying, about searching for juniper.",
```
Comment (scribblecrumb) [inline resources/dicts/patrols/beach/med/greenleaf.json]: You might want to include the cat telling the story within this text.  After all, we want players to be able to tell that the story/lore cat is the one causing this patrol :P so perhaps "As the patrol sets out, p_l remembers an old saying about searching for juniper."

File: resources/dicts/patrols/beach/med/greenleaf.json [hunk 2]
```diff
@@ -5081,6 +5686,77 @@
...
+                "weight": 20,
+                "injury": [
+                    {
+                        "cats": ["r_c"],
+                        "injuries": ["torn pelt"],
+                        "scars": ["SCRATCHSIDE"]
+                    }
+                ],
+                "history_text": {
+                    "scar": "m_c was scarred from bramble thorns on a herb gatherign patrol.",
```
Comment (scribblecrumb) [inline resources/dicts/patrols/beach/med/greenleaf.json]: "gatherign" typo

File: resources/dicts/patrols/beach/med/greenleaf.json (hunk 1 above)
Comment (Tiritirimatangi) [reply resources/dicts/patrols/beach/med/greenleaf.json]: Fixed!

File: resources/dicts/patrols/beach/med/greenleaf.json (hunk 2 above)
Comment (Tiritirimatangi) [reply resources/dicts/patrols/beach/med/greenleaf.json]: Fixed!

File: resources/dicts/patrols/beach/training/any.json [hunk 3]
```diff
@@ -2101,6 +2101,320 @@
...
+           "normal adult": [1, 6]
+            },
+        "weight": 40,
+        "chance_of_success": 60,
+        "pl_skill_constraint": ["STORY,1"],
+        "intro_text": "The sun gleams, splashing orange, yellow, and gold across the sea as it meets the horizon. The seagulls call in the distance, rising on the last thermals of the day, as p_l begins the myth of How Maowi Slowed The Sun.",
+        "decline_text": "A story is nice, but the patrol has to get back to camp, the rest will wait for another time.",
+        "success_outcomes": [
+            {
+                    "text": "Once, the sun raced across the sky, each day only lasting a single catnap. The great Mauwi took his siblings, saying a karakia to StarClan to ask for their help in fighitng the sun. They pinned the great flaming sun in a burrow, clawing at it until it gave in and promised to slow, and this is why the sun gives up its race in leaf-bare and slowes the day, bringing greenleaf.",
```
Comment (scribblecrumb) [inline resources/dicts/patrols/beach/training/any.json]: "fighitng" typo and "slowes" typo

Comment (Tiritirimatangi) [reply resources/dicts/patrols/beach/training/any.json]: fixed!

**Review by Ryos00 — APPROVED** [review]

**Pull request merged**